### PR TITLE
tests: addition of workflow etag tests

### PR DESCRIPTION
**Pull request by drjova**
* Due to the limitation of ``invenio-workflows-ui`` test suite, we're
  adding the tests here to make sure the ``ETag`` is properly set.

Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: tests/integration/workflows/test_views.py [hunk 1]
```diff
@@ -90,3 +91,34 @@ def test_inspect_merge_view_returns_400(workflow_app):
...
+
+        etag = response.headers['ETag']
+        last_modified = response.headers['Last-Modified']
+
+        response = client.get(
+            workflow_url, headers={'If-Modified-Since': last_modified})
+        assert response.status_code == 304
+
+        response = client.get(workflow_url, headers={'If-None-Match': etag})
+        assert response.status_code == 304
```
Comment (david-caro) [inline tests/integration/workflows/test_views.py]: Can you add also one passing the headers but that will return the object?

Comment (drjova) [reply tests/integration/workflows/test_views.py]: do you mean to check the object? on ``304``?

Comment (david-caro) [reply tests/integration/workflows/test_views.py]: no, I mean sending an etag different to the one that's there, and getting 200 with the new object.

Comment (drjova) [reply tests/integration/workflows/test_views.py]: good one, done 👍

**Comment by drjova** [comment]
@david-caro kindly ping

**Comment by drjova** [comment]
ci merge please

**Pull request merged**